From: Sean Wang <sean.wang@mediatek.com>
Date: Fri, 3 Dec 2021 11:30:04 +0800
Subject: Bluetooth: btmtksdio: enable AOSP extension for MT7921
Patch-mainline: v5.17-rc1
Git-commit: 16ada83b9a59a50a6e869d2986fb757007392939
References: jsc#PED-1407

Similar to MT7921 btusb, we enable AOSP extension for MT7921 btmtksdio
as well.

Signed-off-by: Sean Wang <sean.wang@mediatek.com>
Signed-off-by: Marcel Holtmann <marcel@holtmann.org>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/bluetooth/btmtksdio.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/bluetooth/btmtksdio.c
+++ b/drivers/bluetooth/btmtksdio.c
@@ -792,6 +792,7 @@ static int mt79xx_setup(struct hci_dev *
 	}
 
 	hci_set_msft_opcode(hdev, 0xFD30);
+	hci_set_aosp_capable(hdev);
 
 	return err;
 }
